exec/kubernetes_exec_manager.go

```diff
@@ -235,6 +236,45 @@ func (*KubernetesExecManager) Resize(id int, cols uint, rows uint) error {
 	return nil
 }
 
+func (manager *KubernetesExecManager) CreateKubeConfig(token string) error {
+	machineExec := &model.MachineExec{
+		BearerToken: token,
+	}
+	k8sAPI, err := manager.k8sAPIProvider.GetK8sAPI(machineExec)
+	if err != nil {
+		logrus.Debugf("Unable to get k8sAPI %s", err.Error())
+		return err
+	}
+
+	containerFilter := filter.NewKubernetesContainerFilter(manager.namespace, k8sAPI.GetClient().CoreV1())
+	containersInfo, err := containerFilter.GetContainerList()
+	if err != nil {
+		return err
+	}
+
+	if len(containersInfo) == 0 {
+		return errors.New("no containers found to exec")
+	}
+
+	errs := make(map[string]error)
+	for _, containerInfo := range containersInfo {
+
+		cmdResolver := NewCmdResolver(k8sAPI, manager.namespace)
+		_, err := cmdResolver.ResolveCmd(*machineExec, containerInfo)
```

Comment:
Instead of searching for a container with any shell available, I would expect from client to send wanted container name, like via path param like https://github.com/JPinkney/che-machine-exec/compare/init-kubeconfig-rest...sleshchenko:init-kubeconfig-rest-path-param?expand=1

Or even request body: https://github.com/JPinkney/che-machine-exec/compare/init-kubeconfig-rest...sleshchenko:init-kubeconfig-rest-body-param?expand=1 which allows also to propagate the right username to avoid hardcoded developer in kubeconfig.

P.S. The shared examples are not tested